examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/PackageCardAdapter.kt

```diff
@@ -45,26 +45,31 @@ class PackageCardAdapter(
             binding.isActive = activeSubscriptions.contains(product.productId)
 
             binding.packageBuyButton.setOnClickListener {
-                val errorStartingPurchase = validateStartPurchase(product)
-                if (errorStartingPurchase == null) {
-                    listener.onPurchasePackageClicked(
-                        binding.root,
-                        currentPackage,
-                        getSelectedPurchaseOption(),
-                        binding.isUpgradeCheckbox.isChecked
-                    )
-                } else {
-                    showErrorMessage(errorStartingPurchase)
-                }
+                listener.onPurchasePackageClicked(
+                    binding.root,
+                    currentPackage,
+                    binding.isUpgradeCheckbox.isChecked
+                )
             }
 
             binding.productBuyButton.setOnClickListener {
+                listener.onPurchaseProductClicked(
+                    binding.root,
+                    product,
+                    binding.isUpgradeCheckbox.isChecked
+                )
+            }
+
+            binding.optionBuyButton.setOnClickListener {
```

Comment:
We should add to the UI which is the default option and change the label in the other buttons (that buy a package or a product) to indicate it will use the default option. (These improvements can be made in a different PR)

But I think it's great to have the 3 options! It will make testing easier 👍

Reply:
love the idea @tonidero, i'll make a note to add that in my next pr